### PR TITLE
Display name validation is ready.

### PR DESCRIPTION
**Pull request by Osamah-learn**
Display name validation  is ready

1-minimum name letters should be more then 2 and less then 36 
2-accept umlauts letters.
3- smile emotion and special characters are not accepted .

_____________________________________
*-Unit testing

Screenshot 2021-08-19 at 15 12 29

### CODE REVIEW

File: src/lib/__tests__/validators.ts [hunk 1]
```diff
@@ -59,3 +59,37 @@ describe('validateEmail', () => {
...
+describe('Validate The valid displayName to be true', () => {
+  [
+    'Ähtö',
+    'Aki',
+    'Äteritsiputeritsipuolilautatsijänkä',
+    'Veera',
+    'Ivan',
+    'eppu',
+    'feelix',
+    'pyry',
```
Comment (rottabonus) [inline src/lib/__tests__/validators.ts:72]: Probably unnecessary to have all of these when the test-case is basically the same. Maybe add a name with a number, or an allowed special character in the mix.

File: src/lib/__tests__/validators.ts [hunk 2]
```diff
@@ -59,3 +59,37 @@ describe('validateEmail', () => {
...
+describe('Validate The invalid displayName to be true', () => {
+  [
+    'Ähtö@',
+    'Aki©',
+    'Äteritsiputeritsipuolilautatsijänkädsadasd',
+    'Veera..',
+    'Ivan)',
+    'eppu=',
+    'feelix!"',
+    'pyry!"#',
```
Comment (rottabonus) [inline src/lib/__tests__/validators.ts:89]: Maybe add emtpy and too short string here. Probably string with an emoji too?

**Review by rottabonus — APPROVED** [review]
Great, this is good, done similarly than email-validation. Probably could alter the test-cases a bit, but this is good already.

File: src/lib/__tests__/validators.ts (hunk 2 above)
Comment (Osamah-learn) [reply src/lib/__tests__/validators.ts:89]: Ok bro i will thanx for your review

**Pull request merged**